[BUG] - The call is ambiguous between the following methods or properties: 'string.Split(char[]?, StringSplitOptions)' and 'string.Split(string?, StringSplitOptions)'

### Checks

- [x] I have searched the existing issues for a similar bug and it does not exist!

### PlexRipper version

dev

### Platform

Other (Specify in description)

### Browsers

Other (Specify in description)

### Description

When building plexripper in my home-assistant addon the following error arises:

```
  FluentResultExtensions -> /usr/share/plexripper/git/src/FluentResultExtensions/bin/Release/net8.0/FluentResultExtensions.dll
/usr/share/plexripper/git/src/Domain/Extensions/HttpRequestMessageExtensions.cs(21,27): error CS0121: The call is ambiguous between the following methods or properties: 'string.Split(char[]?, StringSplitOptions)' and 'string.Split(string?, StringSplitOptions)' [/usr/share/plexripper/git/src/Domain/Domain.csproj]

Build FAILED.

/usr/share/plexripper/git/src/Domain/Extensions/HttpRequestMessageExtensions.cs(21,27): error CS0121: The call is ambiguous between the following methods or properties: 'string.Split(char[]?, StringSplitOptions)' and 'string.Split(string?, StringSplitOptions)' [/usr/share/plexripper/git/src/Domain/Domain.csproj]
    0 Warning(s)
    1 Error(s)
```

On dotnet SDK 8 using cmd:

`dotnet build --self-contained src/WebAPI/WebAPI.csproj --output /usr/share/plexripper/bin -c Release --ucr=true`

according to AI copilot. the compiler cannot decide between which overload to use. The fix is simple and will follow.



### Reproduction steps

```bash
1. Go to '...'
2. Click on '....'
3. Scroll down to '....'
4. See error
```

### Screenshots

```bash
![DESCRIPTION](LINK.png)
```

### Logs

```bash
7f3aa60d9d31:/usr/share/plexripper/git# dotnet build src/WebAPI/WebAPI.csproj 
-c Release
MSBuild version 17.8.27+3ab07f0cf for .NET
  Determining projects to restore...
  Restored /usr/share/plexripper/git/src/Environment/Environment.csproj (in 854 ms).
  Restored /usr/share/plexripper/git/src/Settings.Contracts/Settings.Contracts.csproj (in 894 ms).
  Restored /usr/share/plexripper/git/src/Data.Contracts/Data.Contracts.csproj (in 893 ms).
  Restored /usr/share/plexripper/git/src/Settings/Settings.csproj (in 894 ms).
  Restored /usr/share/plexripper/git/src/Domain/Domain.csproj (in 898 ms).
  Restored /usr/share/plexripper/git/src/FileSystem.Contracts/FileSystem.Contracts.csproj (in 892 ms).
  Restored /usr/share/plexripper/git/src/Logging/Logging.csproj (in 891 ms).
  Restored /usr/share/plexripper/git/src/Identity.Contracts/Identity.Contracts.csproj (in 894 ms).
  Restored /usr/share/plexripper/git/src/Application.Contracts/Application.Contracts.csproj (in 893 ms).
  Restored /usr/share/plexripper/git/src/FileSystem/FileSystem.csproj (in 900 ms).
  Restored /usr/share/plexripper/git/src/PlexApi.Contracts/PlexApi.Contracts.csproj (in 892 ms).
  Restored /usr/share/plexripper/git/src/FluentResultExtensions/FluentResultExtensions.csproj (in 891 ms).
  Restored /usr/share/plexripper/git/src/WebAPI/WebAPI.csproj (in 900 ms).
  Restored /usr/share/plexripper/git/src/Application/Application.csproj (in 905 ms).
  Restored /usr/share/plexripper/git/src/Data/Data.csproj (in 898 ms).
  Restored /usr/share/plexripper/git/src/PlexApi/PlexApi.csproj (in 897 ms).
  Restored /usr/share/plexripper/git/src/WebAPI.Contracts/WebAPI.Contracts.csproj (in 90 ms).
  Restored /usr/share/plexripper/git/src/Identity/Identity.csproj (in 87 ms).
  Environment -> /usr/share/plexripper/git/src/Environment/bin/Release/net8.0/Environment.dll
  Logging -> /usr/share/plexripper/git/src/Logging/bin/Release/net8.0/Logging.dll
  FluentResultExtensions -> /usr/share/plexripper/git/src/FluentResultExtensions/bin/Release/net8.0/FluentResultExtensions.dll
/usr/share/plexripper/git/src/Domain/Extensions/HttpRequestMessageExtensions.cs(21,27): error CS0121: The call is ambiguous between the following methods or properties: 'string.Split(char[]?, StringSplitOptions)' and 'string.Split(string?, StringSplitOptions)' [/usr/share/plexripper/git/src/Domain/Domain.csproj]

Build FAILED.

/usr/share/plexripper/git/src/Domain/Extensions/HttpRequestMessageExtensions.cs(21,27): error CS0121: The call is ambiguous between the following methods or properties: 'string.Split(char[]?, StringSplitOptions)' and 'string.Split(string?, StringSplitOptions)' [/usr/share/plexripper/git/src/Domain/Domain.csproj]
    0 Warning(s)
    1 Error(s)

Time Elapsed 00:00:09.71
```

--- a/src/Domain/Extensions/HttpRequestMessageExtensions.cs
+++ b/src/Domain/Extensions/HttpRequestMessageExtensions.cs
@@ -18,11 +18,11 @@ public static class HttpRequestMessageExtensions
             query = query.Substring(1);
 
         // Split the query string by '&'
-        var pairs = query.Split(['&'], StringSplitOptions.RemoveEmptyEntries);
+        var pairs = query.Split(new[] { '&' }, StringSplitOptions.RemoveEmptyEntries);
         foreach (var pair in pairs)
         {
             // Split each pair by '='. Use a count of 2 so that values containing '=' are preserved.
-            var parts = pair.Split(['='], 2);
+            var parts = pair.Split(new[] { '=' }, 2);
             if (parts.Length >= 1)
             {
                 // Decode key and value

